vault/templates/statefulset.yaml

```diff
@@ -9,8 +9,9 @@ metadata:
     heritage: {{ .Release.Service }}
 spec:
   replicas: {{ .Values.replicaCount }}
-  strategy:
+  updateStrategy:
     type: {{ .Values.strategy.type }}
+  serviceName: {{ template "vault.name" . }}
```

Comment:
```suggestion
  serviceName: {{ template "vault.fullname" . }}
```

Reply:
@shursun could you please apply this change?